SettingsList.py

```diff
@@ -2744,6 +2744,34 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         choices        = HintDistList(),
         gui_tooltip    = HintDistTips(),
         shared         = True,
+        disable        = {
+            'balanced'  : {'settings' : ['bingosyncURL']},
+            'strong'  : {'settings' : ['bingosyncURL']},
+            'tournament'  : {'settings' : ['bingosyncURL']},
+            'useless'  : {'settings' : ['bingosyncURL']},
+            'very_strong'  : {'settings' : ['bingosyncURL']}
+        },
+    ),
+    Setting_Info(
+        name           ="bingosyncURL",
+        type           =str,
+        choices        ={},
+        gui_type       ="Textinput",
+        gui_text       ="Bingosync URL",
+        shared         = True,
```

Comment:
This needs to be False. `shared` means we'll encode it in the settings string, and we can't do this for freeform text. Does that present an issue for how Bingo is run? Or do you just use the site/patchfiles?

Reply:
We usually just use the site, so it shouldn't be a problem. When playing as a group, usually everyone uses the same board, but the hints will be set in the ROM regardless so it shouldn't be an issue. It's fixed.

Reply:
Sorry, I don't see it. But I'll just fix it when I merge.